src/workers/notifications/squadPostModerationApproved.ts

```diff
@@ -0,0 +1,28 @@
+import { generateTypedNotificationWorker } from './worker';
+import { NotificationType } from '../../notifications/common';
+import { NotificationPostContext } from '../../notifications';
+import { buildPostContext } from './utils';
+
+const worker =
+  generateTypedNotificationWorker<'api.v1.squad-post-moderation-submitted'>({
+    subscription: 'api.v1.squad-post-moderation-submitted-notification',
+    handler: async ({ post: data }, con) => {
+      if (!data?.postId) {
+        return;
+      }
+
+      if (!data?.createdById) {
```

Comment:
What's the use-case where this is null?

Reply:
Just that TS strict thing. Either we early return it or use the `!` notation. I should remove this once the entity is updated from the other PR that `createdById` is NOT nullable anymore.

Reply:
Once entity is not nullable it will auto resolve 👍 Let's avoid `!`

Reply:
Yeah, I tend to avoid them, too, as much as I can.